### ISSUE TITLE
strip shedskin executables by default

### ISSUE THREAD

**Issue by shakfu** [shedskin/shedskin#462]
This is no biggie but it's just something to consider adding at some point ...

On macos `shedskin build --spm test.py` produces an executable of 480K which when stripped via `strip -x test` becomes 363K

On linux  `shedskin build --spm test.py` produces an executable of 3M which when stripped via `strip -s test` becomes 376K

(not yet tested on windows)

It's not such a big win on macOS, but on linux it makes quite a difference.

**Comment by srepmub** [comment]
agreed it would be nice to have an explicit option for enabling/disabling this. not sure if it should be the default though? for example as for bounds-checking, it seems nicer to have the optimization off by default, to be less surprising to the user. I'm guessing that for example --traceback will break after stripping (although we could give an error if the user enables both) . we should also check the documentation to see if the profiling steps still work.

btw I forgot why --traceback is disabled by default. for sure the current code won't work under windows. it also may slow down everything, but not sure. I will create a new issue for this.
